cmd/nodecmd/status.go

```diff
@@ -46,9 +54,58 @@ func statusSubnet(_ *cobra.Command, args []string) error {
 		if blockchainID == ids.Empty {
 			return ErrNoBlockchainID
 		}
-		_, err = getNodeSubnetSyncStatus(blockchainID.String(), clusterName, true, false)
+		notSyncedNodes := []string{}
+		for _, host := range hostAliases {
+			isSubnetSynced, err := getNodeSubnetSyncStatus(blockchainID.String(), clusterName, host, true, false)
+			if err != nil {
+				return err
+			}
+			if !isSubnetSynced {
+				notSyncedNodes = append(notSyncedNodes, host)
+			}
+		}
+		printOutput(hostAliases, notSyncedNodes, clusterName, false)
+		return nil
+	}
+	notBootstrappedNodes, err := checkClusterIsBootstrapped(clusterName)
+	if err != nil {
 		return err
 	}
-	_, err := checkNodeIsBootstrapped(clusterName, true)
-	return err
+	printOutput(hostAliases, notBootstrappedNodes, clusterName, true)
+	return nil
+}
+
+func printOutput(hostAliases, notBootstrappedHosts []string, clusterName string, primaryNetwork bool) {
```

Comment:
let's receive subnetName instead of bool, if subnetName is empty assumes it is primaryNetwork, and we avoid accessing global state also here